src/components/home.css

```diff
@@ -39,20 +65,63 @@ background-color: white;
   */
 .bg-img{
   background-image: linear-gradient( rgba(0, 0, 0, 0.336),rgba(0, 0, 0, 0.281)),
-  url("../../images/fpimg.jpg");
+  url(/images/fpimg.jpg);
   width: 100%;
   height: 100vh;
   background-size: cover;
   color: white;
   text-align: center;
 }
 
-.navLink{
+.wrapper {
+  margin: 0; padding: 0;
+  width: 100%;
+  height: 100vh;
+  background: DimGrey;
+  background-size: cover;
+  font-weight: bolder;
+  font-family: cursive;
+  display: table;
+}
+
+
+.nav {
+  position: fixed;
+  top: 0; left: 0; margin: 0;
+  padding: 0; width: 100%;
+  height: 80px;
+}
+
+img {  
+  margin-right: 50px;
+}
+
+.right {
+  float: right;
+  text-align: right;
+}
+
+.right ul {
+  list-style: none;
+  padding: 1em 3em;
+}
+
+.right ul li {
+  display: inline-block;
+}
+
+.right ul li a {
+  color: white !important;
+  text-decoration: none;
+  font-weight: 500;
+}
+
+.navLink {
   color: #fff !important;
 }
 
 .nav-bar {
-  margin-top: -4.5em;
+  margin-top: -0.7em;
```

Comment:
Why are you using `em` here? Why not just use a fixed `px`

Reply:
Resolved